assets/src/scss/pages/post/_comments-form.scss

```diff
@@ -1,11 +1,10 @@
 .comments-block {
-  .form-section .comment-respond {
```

Comment:
I don't see necessity of looking into the `.form-section`. I think `.comment-responde` is fine.

Reply:
Actually, the reason why we made that difference is because the same `.comment-respond` form is used when replying to a comment, but currently in this case there's no background: see this screenshot from GPI website:

<img width="789" alt="Screenshot 2023-04-21 at 08 57 19" src="https://user-images.githubusercontent.com/6949075/233563585-041ed184-45cb-4f30-be3d-06a80ca14b9d.png">

With this new implementation, this reply form will also have a background color, I think we should check with the design team if they want to make that change:

<img width="777" alt="Screenshot 2023-04-21 at 09 00 19" src="https://user-images.githubusercontent.com/6949075/233563998-3ffc95aa-97ae-427c-8e9c-9b39324e4dde.png">